deploy/helm/charts/templates/node-exporter-service.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.ndmExporter.enabled .Values.ndmExporter.nodeExporter.metricsPort }}
```

Comment:
Why do we need to check for `metricsPort` also for creating the service?

Reply:
Because if the user did not provide the metricsPort or by mistake removed this field then the service creation would fail. It's just to be on a saver side to not get errors.